interfaces/webxr.idl

```diff
@@ -53,15 +63,16 @@ callback XRFrameRequestCallback = void (DOMHighResTimeStamp time, XRFrame frame)
   readonly attribute FrozenArray<XRView> views;
 
   XRDevicePose? getDevicePose(XRCoordinateSystem coordinateSystem);
+  XRInputPose? getInputPose(XRInputSource inputSource, XRCoordinateSystem coordinateSystem);
 };
 
 [SecureContext, Exposed=Window] interface XRCoordinateSystem : EventTarget {
   Float32Array? getTransformTo(XRCoordinateSystem other);
 };
 
 enum XRFrameOfReferenceType {
-  "head-model",
-  "eye-level",
+  "headModel",
```

Comment:
Why is this reverting back to the camelCase variants? https://github.com/immersive-web/webxr/blob/master/explainer.md#appendix-b-proposed-idl uses `head-model` etc

Reply:
Looks like the scrape was out of date, thanks for reviewing thoroughly!